orchestra/static/orchestra/project/js/data_service.js

```diff
@@ -0,0 +1,90 @@
+var serviceModule =  angular.module('orchestra.project.services');
+
+serviceModule.factory('dataService', function() {
+  var dataService = new function() {
+    var that = this;
+    this._meta = {'tasks': {}};
```

Comment:
Fixed!

And it's not actually task data, just metadata about the visualization (for now, just whether the task is expanded or not).